.github/workflows/ci.yml

```diff
@@ -122,15 +122,15 @@ jobs:
       fail-fast: false
       matrix:
         config:
-        - os: ubuntu-20.04
+        - os: ubuntu:18.04
```

Comment:
https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/

Reply:
that's not a github's runner, that's a docker image